README.md

```diff
@@ -12,8 +12,8 @@ Within `nginx/` run
 ```
 ./build.sh
```

Comment:
I guess this line needs to come out since `./build.sh` has been removed, and we should maybe make clear that these instructions are to run the images built by the buildkite scripts.